### PR TITLE
When clicking a deleted type event while it's "deleting", it sends you to 500 error

### PR DESCRIPTION
**Pull request by gitstart**
## What does this PR do?

Fixes #1227
 Loom Video:https://www.loom.com/share/3ed0235b83ef4d64a2f106d514122eb2

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: apps/web/pages/event-types/[type].tsx [hunk 1]
```diff
@@ -1954,6 +1958,7 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
                     {t("delete")}
                   </DialogTrigger>
                   <ConfirmationDialogContent
+                    isLoading={loading}
```
Comment (emrysal) [inline apps/web/pages/event-types/[type].tsx:1961]:
```suggestion
                    isLoading={deleteMutation.isLoading}
```

**Review by emrysal — CHANGES_REQUESTED** [review]
Added a suggestion to turn this into a oneliner fix 👍

**Comment by hariombalhara** [comment]
@gitstart Could you please work on the suggestion and we can quickly merge it.

**Comment by gitstart** [comment]
sorry for the delay @hariombalhara, we will fix the notification issue with the bot that made us miss the comments

the change is pushed

**Pull request closed without merging**